src/lib/network/install_inf_convertor.rb

```diff
@@ -263,139 +99,8 @@ def write_proxy
-        BASH_PATH,
-        "sed -i s/^WAIT_FOR_INTERFACES=.*/WAIT_FOR_INTERFACES=#{connect_wait}/g /etc/sysconfig/network/config"
-      )
-
-      ret == 0
-    end
-
-    def dev_name
-      netdevice = InstallInf["Netdevice"].to_s
-
-      log.info("InstInstallInfClient#dev_name:#{netdevice}")
-
-      netdevice
-    end
-
     def stdout_of(command)
       SCR.Execute(path(".target.bash_output"), command)["stdout"].to_s
     end
-
-    def create_wlan_ifcfg
-      wlan_key = InstallInf["WlanKey"]
-      wlan_essid = InstallInf["WlanESSID"]
-      wlan_key_type = InstallInf["WlanKeyType"]
-      wlan_auth = InstallInf["WlanAuth"]
-      wlan_key_len = InstallInf["WlanKeyLen"]
-
-      return "" if wlan_essid.empty?
-
-      ifcfg = "WIRELESS_ESSID='#{wlan_essid}'\n"
-
-      case wlan_auth
-      when "", "psk"
-        ifcfg << "WIRELESS_WPA_PSK='#{wlan_key}'\n"
-        ifcfg << "WIRELESS_AUTH_MODE='psk'\n"
-
-      when "open"
-        ifcfg << "WIRELESS_AUTH_MODE='no-encryption'\n"
-
-      when "wep_open", "wep_restricted"
-        type = if wlan_key_type == "password"
-          "h:"
-        elsif wlan_key_type == "ascii"
-          "s:"
-        else
-          wlan_key_type[0] + ":"
-        end
-
-        wlan_auth_mode = wlan_auth == "wep-open" ? "open" : "sharedkey"
-
-        ifcfg << "WIRELESS_AUTH_MODE='#{wlan_auth_mode}'\n"
-        ifcfg << "WIRELESS_DEFAULT_KEY='0'\n"
-        ifcfg << "WIRELESS_KEY_0='#{type}#{wlan_key}'\n"
-        ifcfg << "WIRELESS_KEY_LENGTH='#{wlan_key_len}'\n"
-      end
-
-      ifcfg
-    end
-
-    def create_s390_ifcfg(hardware)
-      hwaddr = InstallInf["OSAHWAddr"]
-      return "" if hwaddr.empty?
-
-      netdevice = dev_name
-      return "" if netdevice.empty?
-
-      devtype = NetworkInterfaces.GetType(netdevice)
-
-      log.info("Interface type: #{devtype}")
-
-      # only some card types need a persistent MAC (bnc#658708)
-      sysfs_id = dev_name_to_sysfs_id(netdevice, hardware)
-      return "" if !s390_device_needs_persistent_mac(sysfs_id, hardware)
-
-      # hsi devices do not support setting hwaddr (bnc #479481)
-      return "" if devtype == "hsi"
-
-      # set HW address only for qeth set to Layer 2 (bnc #479481)
-      return "" if devtype == "eth" && InstallInf["Layer2"] != "1"
-
-      "LLADDR='#{hwaddr}'\n"
-    end
-
-    def create_device_name_ifcfg(hardware)
-      device_name = dev_name
-
-      # authoritative sources of device name are:
-      # - hwinfo
-      # - install.inf
-      # nobody else was able to edit device name so far (so ifcfg["NAME"])
-      # is empty
-      hw_name = HardwareName(hardware, device_name)
-      hw_name = InstallInf["NetCardName"] || "" if hw_name.empty?
-
-      return "" if hw_name.empty?
-
-      # protect special characters, #305343
-      "NAME='#{String.Quote(hw_name)}'\n"
-    end
-
-    def write_ifcfg(ifcfg)
-      device_name = dev_name
-
-      return false if device_name.empty?
-
-      if !AllowUdevModify()
```

Comment:
Never executed because device_name is always empty, but still used by `LanItems`